docs/Drawer.md

```diff
@@ -263,176 +280,184 @@ Any other props supplied will be provided to the root element (`div`).
+| hidden                          | Sets whether to hide the nav item                                                                                                                           | `boolean`               | no       |         |
+| hidePadding                     | Sets whether to collapse the left padding when no icon is used                                                                                              | `boolean`               | no       |         |
+| icon                            | A component to render for the left icon                                                                                                                     | `JSX.Element`           | no       |         |
+| isInActiveTree*                 | Sets whether the item is a parent of the currently active item (managed automatically)                                                                      | `boolean`               | no       |         |
+| itemID                          | An unique identifier of the NavItem. Item will have 'active' style when this matches activeItem                                                             | `string`                | yes      |         |
+| items                           | The items nested under this item                                                                                                                            | `NestedNavItem[]`       | no       |         |
+| notifyActiveParent*             | Callback function to the parent element to update active hierarchy styles                                                                                   | `(ids: string[]) => void` | no     |         |
+| onClick                         | A function to execute when clicked                                                                                                                          | `function`              | no       |         |
+| rightComponent                  | An icon/component to display to the right                                                                                                                   | `ReactNode`             | no       |         |
+| statusColor                     | Status stripe and icon color                                                                                                                                | `string`                | no       |         |
+| subtitle                        | The text to show on the second line                                                                                                                         | `string`                | no       |         |
+| title                           | The text to show on the first line                                                                                                                          | `string`                | yes      |         |
```

Comment:
`title` is now no longer required, `icon` is

Reply:
That's backwards. For the NavItem, the title is required and icon is optional. For RailItem, they are both optional.